code/modules/clothing/spacesuits/rig/modules/utility.dm

```diff
@@ -669,14 +666,18 @@
 			if (combatType && ismob(aa))
 				continue
 
-			if (aa.density)
+			if (aa.density && NOT_FLAG(aa.flags, ON_BORDER))
```

Comment:
I am unsure if this is sufficient or if there should also be a check for the location of the railing.
(So you can only leap to a tile, if you dont have to leap over a railing)

Reply:
in the case of a railing: it seems you can leap over railings with the leg actuator
in cases where you can't leap over something: you slam into it, which is fine, imo
![image](https://user-images.githubusercontent.com/22774890/221360176-7369cd0f-648e-4788-a6c0-dbb096de4a0f.png)